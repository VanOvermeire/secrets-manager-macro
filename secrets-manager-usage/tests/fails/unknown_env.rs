use secrets_manager_macro::build_secrets_struct;

#[build_secrets_struct(envs = dev,fake)]
struct SecretManagerTestSecret {}

fn main() {}
