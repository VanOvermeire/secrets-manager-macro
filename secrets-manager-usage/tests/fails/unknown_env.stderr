error: received envs dev,fake but only matched these secrets: /dev/secrets-manager-test-secret
 --> tests/fails/unknown_env.rs:4:8
  |
4 | struct SecretsManagerTestSecret {}
  |        ^^^^^^^^^^^^^^^^^^^^^^^^
